core/src/main/java/org/lflang/generator/ReactionInstance.java

```diff
@@ -29,6 +29,7 @@
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Set;
+import org.eclipse.xtext.xbase.lib.StringExtensions;
```

Comment:
Is this import really necessary?  It looks like it is used to just check for a null or empty string. I think that this is a rather heavyweight dependency for such small functionality.

Reply:
We already have this dependency throughout the code base. If we want to remove it, we have quite a bit more work to do...

Reply:
Xtext is one of our core dependencies. Using its functionality throughout shouldn't do any additional harm.